### PR TITLE
Building ulm-wasm using kdist

### LINKED ISSUE #44: Add ulm-wasm as a Target in the `kdist` plugin
Currently ulm-wasm is built using make. The main aim of this issue is to manage this process using the kdist plugin
A secondary goal is to familiarize with the kdist build system.

Given the secondary goal, my estimate is that it might take me a couple of days to figure it out.

### PR DESCRIPTION
**Pull request by traiansf**
Building `ulm-wasm` using `kdist` (only the K part for now; dependencies are still built through make)

### CODE REVIEW

File: pykwasm/src/pykwasm/kdist/wasm-semantics/ulm-wasm.md [hunk 1]
```diff
@@ -180,8 +180,8 @@ In the standalone semantics, the Wasm VM obtains an entrypoint from the configur
 
 ```local
     syntax WasmString ::= #getEntryPoint() [function, total]
-    rule #getEntryPoint() => FUNCNAME
-         [[ <entry> FUNCNAME </entry> ]]
+    rule [[ #getEntryPoint() => FUNCNAME ]]
```
Comment (sskeirik) [inline pykwasm/src/pykwasm/kdist/wasm-semantics/ulm-wasm.md:187]: Wow. I noticed this failed to compile and I kept trying different permutations of config cells, moving the configuration declaration around, etc... trying to get it to run. Somehow I forgot that the rule is bracketed and not the configuration cells. 😵

File: pykwasm/src/pykwasm/kdist/plugin.py [hunk 2]
```diff
@@ -55,6 +55,48 @@ def deps(self) -> tuple[str]:
         return ('wasm-semantics.source',)
 
 
+def ulm_wasm_args(src_dir: Path, ulm_test: bool = False) -> dict[str, Any]:
+    build_dir = Path(__file__).parent.parent.parent.parent.parent / 'build'
+    print(build_dir)
```
Comment (sskeirik) [inline pykwasm/src/pykwasm/kdist/plugin.py]: I'd remove this print statement unless we have a separate flag for debugging the build script to gate it behind.

**Review by sskeirik — APPROVED** [review]
This looks good! Thanks! If you could take a look at one minor comment, that would be awesome.

File: pykwasm/src/pykwasm/kdist/plugin.py (hunk 2 above)
Comment (traiansf) [reply pykwasm/src/pykwasm/kdist/plugin.py]: sorry. I had forgotten about it.

File: pykwasm/src/pykwasm/kdist/wasm-semantics/ulm-wasm.md (hunk 1 above)
Comment (traiansf) [reply pykwasm/src/pykwasm/kdist/wasm-semantics/ulm-wasm.md:187]: well, it was actually @virgil-serbanuta who knew the right incantation :-)

**Pull request merged**